docs/modules/ROOT/pages/how-tos/workflows/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,107 @@
+= Maintaining remote Tekton pipelines up to date
+
+[IMPORTANT]
+====
+* This article took for granted the reader knows what a link:https://tekton.dev/docs/pipelines/resolution/[Tekton remote pipeline] is. If this is not the case, please read more about this feature before proceeding.
+====
+
+== The advantage of using a remote pipeline
+
+This approach allows a team or an individual contributor to manage plenty of Pipelines from a single repository as source of truth. But there also other good reasons:
+
+. Pipelines are stored in a Git repository and fetched during execution, eliminating the need to maintain local or static pipeline definitions.
+. Teams can leverage Git’s versioning to access specific pipeline branches
+. In case of issues, it's very simple to rollback to a previous pipeline version, and apply it globally among the Konflux Components.
```

Comment:
It is possible to roll back to previous versions without this remote pipeline strategy as well. I guess that the suggestion is to rollback the reference pipeline commit based on the use of a floating branch tag? If you are suggesting that it is easy to roll back, I think we should document that in this page.

Reply:
> I guess that the suggestion is to rollback the reference pipeline commit based on the use of a floating branch tag?
Yup, that's exactly what I meant. Let me specify it better

Reply:
I was not thinking about using a floating branch tag, I have not tested it despite I suppose it works out of the box. I would leave it as it is and we could always edit this page in the future eventually to add more details or strategies.